docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -59,6 +59,45 @@ the example below.
 
 The exact format description is described in the SQLAlchemy documentation, see `Database Urls <https://docs.sqlalchemy.org/en/14/core/engines.html>`__. We will also show you some examples below.
 
+SQLite database setup
+---------------------
+
+SQLite database can be used to run Airflow for development purpose as it does not require any database server
```

Comment:
Can you add docs about connection URI?  The sections for other databases cover at least one example of such a URI, so adding an example in this section would be helpful.

Reply:
Good point. Added.